rpcs3/Emu/Cell/lv2/sys_ss.cpp

```diff
@@ -124,19 +124,16 @@ error_code sys_ss_access_control_engine(u64 pkg_id, u64 a2, u64 a3)
 	return CELL_OK;
 }
 
-s32 sys_ss_get_console_id(vm::ptr<u8> buf)
+error_code sys_ss_get_console_id(vm::ptr<u8> buf)
 {
-	sys_ss.todo("sys_ss_get_console_id(buf=*0x%x)", buf);
+	sys_ss.trace("sys_ss_get_console_id(buf=*0x%x)", buf);
 
-	// TODO: Return some actual IDPS?
-	*buf = 0;
-
-	return CELL_OK;
+	return sys_ss_appliance_info_manager(0x19003, buf);
 }
 
-s32 sys_ss_get_open_psid(vm::ptr<CellSsOpenPSID> psid)
+error_code sys_ss_get_open_psid(vm::ptr<CellSsOpenPSID> psid)
 {
-	sys_ss.warning("sys_ss_get_open_psid(psid=*0x%x)", psid);
+	sys_ss.trace("sys_ss_get_open_psid(psid=*0x%x)", psid);
```

Comment:
all

Reply:
Just wonder whether the other two sys_ss syscalls are important enough to be logged clearly.

Reply:
When VSH starts to write GBs of log I'll consider it.

Reply:
Alright.